tests/baselines/reference/tsserver/projectReferences/find-refs-to-decl-in-other-proj-when-proj-is-loaded-and-refd-proj-loading-is-disabled-and-proj-ref-redirects-are-disabled-and-a-decl-map-is-present.js

```diff
@@ -122,7 +122,5 @@ Finding references to /user/username/projects/myproject/a/index.ts position 40 i
 FileWatcher:: Added:: WatchInfo: /user/username/projects/myproject/b/lib/index.d.ts.map 500 undefined WatchType: Closed Script info
 Search path: /user/username/projects/myproject/b
 For info: /user/username/projects/myproject/b/index.ts :: Config file name: /user/username/projects/myproject/b/tsconfig.json
-Search path: /user/username/projects/myproject/b

```

Comment:
A pair of lines like this indicates that we were walking up from (e.g.) `/user/username/projects/myproject/b/index.ts` to find the nearest (aka "default"?) tsconfig, (e.g.) `/user/username/projects/myproject/b/tsconfig.json`.  If you scroll up, you will find an identical pair of lines indicating that this search was redundant.